### PR TITLE
test: [#187091927] cypress poppy onboarding mobile

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

https://www.pivotaltracker.com/story/show/186966376
### Approach
Add cypress mobile test for 1 industry/ starting a business. We also created specific describe blocks for desktop and mobile to improve readability

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/cypress/e2e/onboarding-as-starting-business.spec.ts [hunk 1]
```diff
@@ -1,84 +1,109 @@
...
+import { carServiceOptions, Industries, Industry, randomInt } from "@businessnjgovnavigator/shared/";
 
 const enabledIndustries = Industries.filter((element: Industry) => {
   return element.isEnabled;
 });
 
 describe("Onboarding for all industries when starting a business [feature] [all] [group4]", () => {
-  beforeEach(() => {
-    cy.loginByCognitoApi();
+  describe("Desktop", () => {
```
Comment (FarazA22) [inline web/cypress/e2e/onboarding-as-starting-business.spec.ts:10]: wrapped in a describe block, no other changes

File: web/cypress/e2e/onboarding-as-starting-business.spec.ts [hunk 2]
```diff
@@ -1,84 +1,109 @@
...
+        onOnboardingPageStartingBusiness.clickNext();
+        cy.url().should("include", "dashboard?fromOnboarding=true");
+        cy.get('[data-testid="header-link-to-profile"]');
+      });
+    }
   });
 
-  for (const industry of enabledIndustries) {
-    it(`Onboarding for ${industry.name}`, () => {
+  describe("Mobile", () => {
```
Comment (FarazA22) [inline web/cypress/e2e/onboarding-as-starting-business.spec.ts:89]: second describe block for mobile, entire block is new

**Review by adidner — APPROVED** [review]
LGMT

**Review by jphechter — APPROVED** [review]
I think everything here makes sense. Nice work 👍

**Pull request merged**